### DISCUSSION TITLE
قطع شدن کانکشن ها روی نت ایرانسل

### DISCUSSION THREAD

**Discussion by samadblaj** [Gozargah/Marzban#613] [General]
سلام دوستان
از پنل مرزبان استفاده میکنیم ip مسدود نشده.
 اما توی خیلی از شهرها همه افرادی که نت ایرانسل داشتن قطع شدن راهکار چیه؟

**Comment by DeAref** [comment]
کانفیگ xray رو هم کامنت کن
